addons/l10n_in_pos/static/src/overrides/components/pos_receipt.xml

```diff
@@ -26,6 +26,29 @@
                 </div>
             </t>
         </xpath>
+        <xpath expr="//div[hasclass('pos-receipt-taxes')]" position="replace">
+            <div t-if="props.data.tax_details.length > 0" class="pos-receipt-taxes justify-content-between">
+                <span>GST%</span>
+                <span>SGST</span>
+                <span>CGST</span>
+                <span>ExGST</span>
+                <span>Total</span>
+                <t t-foreach="props.data.tax_details" t-as="tax" t-key="tax.tax.id">
+                    <span><t t-esc="tax.tax.amount"/> %</span>
+                    <span t-esc="props.formatCurrency(tax.amount / 2, false)" />
+                    <span t-esc="props.formatCurrency(tax.amount / 2, false)" />
+                    <span t-esc="props.formatCurrency(tax.base, false)" />
+                    <span t-esc="props.formatCurrency(tax.amount + tax.base, false)" />
+                </t>
+                <t t-if="props.data.tax_details.length > 1">
+                    <span />
+                    <span t-esc="props.formatCurrency(props.data.amount_tax / 2, false)" />
+                    <span t-esc="props.formatCurrency(props.data.amount_tax / 2, false)" />
+                    <span t-esc="props.formatCurrency(props.data.total_without_tax, false)" />
+                    <span t-esc="props.formatCurrency(props.data.amount_total, false)" />
+                </t>
+            </div>
+        </xpath>
```

Comment:
There is a similarly looking override below. Can you tell me the difference? Can you put a screenshot in the PR description?

Reply:
@caburj The below one is for HSN Summary ( which summarize GST taxes as per HSN code)
This task is for to change the tax name from vat to GST, I guess can we do it for standard?
In India we have GST and similarly other country can have different tax names according to the localization 

What is your opinion?

![image](https://github.com/odoo/odoo/assets/121081873/059a7232-3a08-449b-a615-7c6d642683b5)